package.json

```diff
@@ -201,7 +201,7 @@
     "socket.io": "^4.6.0",
     "socket.io-client": "^4.6.0",
     "sqs-consumer": "5.7.0",
-    "sqs-consumer-v6": "npm:sqs-consumer@^6.2.0",
+    "sqs-consumer-v7": "npm:sqs-consumer@^7.4.0",
```

Comment:
Question: We are testing aws-sdk v2 with the `sqs-consumer` package (see the test in aws-sdk v2 sqs/sqs-consumer). I thought sqs-consumer v7 is using aws-sdk v3 under the hood.

Reply:
Ignore the question. I thought we are only installing one sqs-consumer version now.
The packages needed to be switched though 👍